ivy_tests/test_ivy/test_frontends/test_torch/test_Utilities.py

```diff
@@ -0,0 +1,80 @@
+# global
```

Comment:
File names should be all lowercase, matching the format of the rest of the code base.

Reply:
Again, this file needs to be renamed to be all lowercase `test_utilities.py` not `test_Utilities.py`